en/flight_modes_mc/throw_launch.md

```diff
@@ -0,0 +1,97 @@
+# Throw Launch (Multicopter)
+
+:::warning Experimental
+This feature was introduced after PX4 v1.14.
+
+- It has not yet been broadly tested on different vehicle configurations or scenarios.
+- The majority of testing has been done in position mode.
+  Other modes should also work.
+
+:::
+
+This feature allows a multicopter to be started by arming it from a fixed position and then throwing it into the air.
+The vehicle then turns on the motors and operates according to its current mode.
+
+When throw launch is enabled, arming the vehicle does not cause the propellers to spin.
+The propellors will not activate until the vehicle is thrown or is disarmed, and the arming tone will continue playing during this time.
+The vehicle will not automatically disarm after arming, and must be manually disarmed if you choose not to throw it.
+
+The vehicle detects that it has been thrown based on reaching a certain speed (5m/s), and then starts the motors at the apex of the throw (once it determines that it has started to fall).
+You need to throw the vehicle high enough so that it can stabilize its height well before falling anywhere near people or obstacles.
+
+Notes:
+
+- The mode is disabled by default, and must be enabled using a [parameter](#parameters) before arming.
+- When enabled you cannot take off from the ground using the normal modes.
+- The vehicle should not be transported after being armed and before the throw.
+  In particular, the throw should not be executed from a moving platform.
+  The reason for this is that the condition to start the motors depends on absolute speed of the multicopter and does not account for any additional movement.
+  Trying to throw the drone from a moving platform might result in the motors being started prematurely.
```

Comment:
@MaEtUgR OK, I rewrote this a little to ensure it mentions the arming tone, and that the vehicle must be enabled using a parameter, and that you can't take off from the ground using normal modes. 

This should all be about right now. If no comments will merge tomorrow.